<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyAcc</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resourse.qrc">
    <normaloff>:/photo/lighthouse.ico</normaloff>:/photo/lighthouse.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QToolButton" name="addData">
    <property name="geometry">
     <rect>
      <x>674</x>
      <y>10</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить данные</string>
    </property>
   </widget>
   <widget class="QToolButton" name="update">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>25</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputURL">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>441</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>116</x>
      <y>50</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите URl-адрес:</string>
    </property>
   </widget>
   <widget class="QToolButton" name="go">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>50</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>791</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>821</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="file">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="login"/>
    <addaction name="account_setting"/>
    <addaction name="separator"/>
    <addaction name="check_update"/>
   </widget>
   <widget class="QMenu" name="edit">
    <property name="title">
     <string>Правка</string>
    </property>
    <addaction name="unhide"/>
    <addaction name="hide"/>
    <addaction name="separator"/>
    <addaction name="add_data"/>
    <addaction name="change_data"/>
   </widget>
   <widget class="QMenu" name="perform_backup">
    <property name="title">
     <string>Резервное восстановление</string>
    </property>
   </widget>
   <widget class="QMenu" name="view">
    <property name="title">
     <string>Вид</string>
    </property>
   </widget>
   <widget class="QMenu" name="help">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="support"/>
    <addaction name="about"/>
   </widget>
   <addaction name="file"/>
   <addaction name="edit"/>
   <addaction name="perform_backup"/>
   <addaction name="view"/>
   <addaction name="help"/>
  </widget>
  <action name="login">
   <property name="icon">
    <iconset resource="resourse.qrc">
     <normaloff>:/photo/Авторизация.ico</normaloff>:/photo/Авторизация.ico</iconset>
   </property>
   <property name="text">
    <string>Вход в систему</string>
   </property>
  </action>
  <action name="account_setting">
   <property name="text">
    <string>Настройки учетной записи</string>
   </property>
  </action>
  <action name="check_update">
   <property name="text">
    <string>Проверка обновлений</string>
   </property>
  </action>
  <action name="unhide">
   <property name="text">
    <string>Отобразить все списки</string>
   </property>
  </action>
  <action name="hide">
   <property name="text">
    <string>Скрыть все списки</string>
   </property>
  </action>
  <action name="change_data">
   <property name="text">
    <string>Изменить данные о ...</string>
   </property>
  </action>
  <action name="support">
   <property name="icon">
    <iconset resource="resourse.qrc">
     <normaloff>:/photo/support.ico</normaloff>:/photo/support.ico</iconset>
   </property>
   <property name="text">
    <string>Поддержка</string>
   </property>
  </action>
  <action name="about">
   <property name="icon">
    <iconset resource="resourse.qrc">
     <normaloff>:/photo/info.ico</normaloff>:/photo/info.ico</iconset>
   </property>
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="add_data">
   <property name="icon">
    <iconset resource="resourse.qrc">
     <normaloff>:/photo/add.ico</normaloff>:/photo/add.ico</iconset>
   </property>
   <property name="text">
    <string>Добавить данные</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resourse.qrc"/>
 </resources>
 <connections/>
</ui>
